include/deal.II/numerics/adaptation_strategies.h

```diff
@@ -0,0 +1,355 @@
+     * child.
+     *
+     * @f[
+     *   d_{K_c} = d_{K_p}
+     *   \qquad
+     *   \forall K_c \text{ children of } K_p
+     * @f]
+     */
+    template <int dim, int spacedim, typename value_type>
+    std::vector<value_type>
+    preserve(const typename dealii::Triangulation<dim, spacedim>::cell_iterator
+               &              parent,
+             const value_type parent_value);
+
+    /**
+     * Return a vector which contains data of the parent cell being equally
+     * divided among all children.
+     *
+     * @f[
+     *   d_{K_c} = d_{K_p} / n_\text{children}
+     *   \qquad
+     *   \forall K_c \text{ children of } K_p
+     * @f]
+     *
+     * This strategy preserves the $l_1$-norm of the corresponding global data
+     * Vector before and after adaptation.
+     */
+    template <int dim, int spacedim, typename value_type>
+    std::vector<value_type>
+    split(const typename dealii::Triangulation<dim, spacedim>::cell_iterator
+            &              parent,
+          const value_type parent_value);
+
+    /**
+     * Return a vector which contains squared data of the parent cell being
+     * equally divided among the squares of all children.
+     *
+     * @f[
+     *   d_{K_c}^2 = d_{K_p}^2 / n_\text{children}
+     *   \qquad
+     *   \forall K_c \text{ children of } K_p
+     * @f]
+     *
+     * This strategy preserves the $l_2$-norm of the corresponding global data
+     * Vector before and after adaptation.
+     */
+    template <int dim, int spacedim, typename value_type>
+    std::vector<value_type>
+    l2_norm(const typename dealii::Triangulation<dim, spacedim>::cell_iterator
+              &              parent,
+            const value_type parent_value);
+  } // namespace Refinement
+
+  /**
+   * For coarsening, all strategies take the parent cell and a vector of data
+   * that belonged to its former children. They return the value that will be
+   * assigned to the parent cell.
+   *
+   * The ordering of values in the vector for children data corresponds to the
+   * index when calling TriaAccessor::child_index.
+   */
+  namespace Coarsening
+  {
+    /**
+     * Check if data on all children match, and return value of the first child.
+     *
+     * @f[
+     *   d_{K_p} = d_{K_c}
+     *   \qquad
+     *   \forall K_c \text{ children of } K_p
+     * @f]
+     */
+    template <int dim, int spacedim, typename value_type>
+    value_type
+    check_equality(
+      const typename dealii::Triangulation<dim, spacedim>::cell_iterator
+        &                            parent,
+      const std::vector<value_type> &children_values);
+
+    /**
+     * Return sum of data on all children.
+     *
+     * @f[
+     *   d_{K_p} = \sum d_{K_c}
+     *   \qquad
+     *   \forall K_c \text{ children of } K_p
+     * @f]
+     *
+     * This strategy preserves the $l_1$-norm of the corresponding global data
+     * Vector before and after adaptation.
```

Comment:
```suggestion
     * vector before and after adaptation.
```
I see no need to capitalize this one.